_topic_maps/_topic_map.yml

```diff
@@ -600,47 +600,50 @@ Topics:
-  File: migrating-to-multi-payload
-- Name: Performing update using canary rollout strategy
-  File: update-using-custom-machine-config-pools
-- Name: Updating a cluster that includes RHEL compute machines
-  File: updating-cluster-rhel-compute
-  Distros: openshift-enterprise
-- Name: Updating a cluster in a disconnected environment
-  Dir: updating-restricted-network-cluster
-  Topics:
-  - Name: About cluster updates in a disconnected environment
-    File: index
-  - Name: Mirroring the OpenShift Container Platform image repository
-    File: mirroring-image-repository
-  - Name: Updating a cluster in a disconnected environment using OSUS
-    File: restricted-network-update-osus
+- Name: Performing a cluster update
+  Dir: updating_a_cluster
+  Topics:
+  - Name: Updating a cluster using the CLI
+    File: updating-cluster-cli
+  - Name: Updating a cluster using the web console
+    File: updating-cluster-web-console
+  - Name: Performing an EUS-to-EUS update
+    File: eus-eus-update
     Distros: openshift-enterprise
-  - Name: Updating a cluster in a disconnected environment without OSUS
-    File: restricted-network-update
-    Distros: openshift-enterprise
-  - Name: Updating a cluster in a disconnected environment by using the CLI
-    File: restricted-network-update
-    Distros: openshift-origin
-  - Name: Uninstalling OSUS from a cluster
-    File: uninstalling-osus
+  - Name: Performing a canary rollout update
+    File: update-using-custom-machine-config-pools
+  - Name: Updating a cluster that includes RHEL compute machines
+    File: updating-cluster-rhel-compute
     Distros: openshift-enterprise
-- Name: Updating hardware on nodes running on vSphere
-  File: updating-hardware-on-nodes-running-on-vsphere
+  - Name: Updating a cluster in a disconnected environment
+    Dir: updating_disconnected_cluster
+    Topics:
+    - Name: About cluster updates in a disconnected environment
+      File: index
+    - Name: Mirroring OpenShift Container Platform images
+      File: mirroring-image-repository
+    - Name: Updating a cluster in a disconnected environment using OSUS
+      File: disconnected-update-osus
+      Distros: openshift-enterprise
+    - Name: Updating a cluster in a disconnected environment without OSUS
+      File: disconnected-update
+      Distros: openshift-enterprise
+    - Name: Updating a cluster in a disconnected environment by using the CLI
+      File: disconnected-update
+      Distros: openshift-origin
+    - Name: Uninstalling OSUS from a cluster
+      File: uninstalling-osus
+      Distros: openshift-enterprise
+  - Name: Updating hardware on nodes running on vSphere
+    File: updating-hardware-on-nodes-running-on-vsphere
+  - Name: Migrating to a cluster with multi-architecture compute machines
+    File: migrating-to-multi-payload
+  - Name: Updating hosted control planes
+    File: updating-hosted-control-planes
+- Name: Preparing to update a cluster with manually maintained credentials
```

Comment:
Just checking if these two sections should go before the performing the update subsection, since they're preparing sections?

Reply:
So the plan with this overall reorg is to put every assembly in 4.14+ update docs into a subdirectory. These two preparing sections will soon be put into a subsection titled something like "Preparing to update your cluster", which will be placed above this new "updating" subsection, and the plan is to have the entire reorg complete before 4.14 GA.

Let me know if you'd prefer me to move these preparing sections in this PR just to be safe, but these will otherwise be accounted for soon.

Reply:
Nope that's totally fine to leave them as is for now then, thanks for the explanation!